torch_geometric/nn/meta.py

```diff
@@ -104,9 +106,10 @@ def reset_parameters(self):
                 item.reset_parameters()
 
     def forward(
-            self, x: Tensor, edge_index: Tensor,
-            edge_attr: Optional[Tensor] = None, u: Optional[Tensor] = None,
-            batch: Optional[Tensor] = None) -> Tuple[Tensor, Tensor, Tensor]:
+        self, x: Tensor, edge_index: Tensor,
+        edge_attr: Optional[Tensor] = None, u: Optional[Tensor] = None,
+        batch: Optional[Tensor] = None
+    ) -> Tuple[Optional[Tensor], Optional[Tensor], Optional[Tensor]]:
```

Comment:
```suggestion
    ) -> Tuple[Tensor, Optional[Tensor], Optional[Tensor]]:
```

Reply:
Hi! I agree it makes more sense to remove the `Optional[]`. However, current `test_meta_layer` test fails with this type hinting, as we use a dummy model for `node_model` which always return `None` (and thus returned `x` value is `None` in that case).
Maybe I could update the dummy model to return `edge_attr` for `edge_model`, `x` for `node_model` and `u` for `global_model` ?

Reply:
Yeah, it probably makes sense to define them separately with the correct type hints.